src/Raven.Client/ServerWide/Operations/Configuration/ServerWideBackupConfiguration.cs

```diff
@@ -6,7 +6,8 @@ namespace Raven.Client.ServerWide.Operations.Configuration
     public class ServerWideBackupConfiguration : PeriodicBackupConfiguration
     {
         internal static string NamePrefix = "Server Wide Backup";
-        
+        internal string ToRemoveFromDatabaseRecord { get; set; }
```

Comment:
Why do we need this? How do we handle update of a name of regular (not server wide) backup tasks?

Reply:
Additionally it's hard to know what is it actually about - is about TaskId to remove or TaskName to remove.

Reply:
In the database record, the `TaskId` is constant while in the server-wide configuration the `TaskId` change on any update to the configuration so we can't use the `TaskId` from the server-wide to know which configuration to edit (remove) in the database record (usually, we use the name but it is not useful when the user edits the name).

One option is to change this behavior and make the `TaskId` of server-wide backup constant as well.
This kind of change will not "help" to already exist server-wide backup configuration which already has different `TaskId` from database record `TaskId`. In this solution, we maybe can use a scheme upgrade.

In the current solution, I am using the name from the server-wide configuration update. This doesn't need any scheme upgrade

Reply:
Do I understand correctly that the server wide backup configuration has _constant_ `TaskId` in the database record? Can you please point me to the relevant code? Why is that?

@grisha-kotler What do you think?

Reply:
Here https://github.com/ravendb/ravendb/pull/11086/files#diff-ad98abd3baf8509e485969ace15cd336R3098
```
if (backup.TryGet(nameof(PeriodicBackupConfiguration.TaskId), out long taskId))
                                    oldTaskId = taskId;
```